Gemfile.lock

```diff
@@ -142,6 +147,10 @@ GEM
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     ffi (1.17.0)
+    ffi (1.17.0-aarch64-linux-musl)
+    ffi (1.17.0-arm64-darwin)
+    ffi (1.17.0-x86_64-darwin)
+    ffi (1.17.0-x86_64-linux-gnu)
```

Comment:
Oh shoot, I don't think you mean to change these but I believe we have another PR in flight to fix. I'll make sure to get that merged in ahead of your PR.